processor/resourceprocessor/resource_processor_test.go

```diff
@@ -18,78 +18,33 @@ import (
 	"context"
 	"testing"
 
-	resourcepb "github.com/census-instrumentation/opencensus-proto/gen-go/resource/v1"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
 	"go.opentelemetry.io/collector/config/configmodels"
-	"go.opentelemetry.io/collector/consumer/consumerdata"
+	"go.opentelemetry.io/collector/consumer/pdata"
+	"go.opentelemetry.io/collector/consumer/pdatautil"
+	"go.opentelemetry.io/collector/internal/data/testdata"
 )
 
 var (
-	cfg = &Config{
-		ProcessorSettings: configmodels.ProcessorSettings{
-			TypeVal: "resource",
-			NameVal: "resource",
-		},
-		ResourceType: "host",
```

Comment:
Same here: please replace this with `"opencensus.type": "host"` attribute rather than deleting it.

Reply:
added the backward compatibility